### PR TITLE
fix the content order issue and typo mistake

### PR DESCRIPTION
**Pull request by loka-capco**

### CODE REVIEW

File: src/containers/CategoryPage/CategoryPage.tsx [hunk 1]
```diff
@@ -289,44 +289,50 @@ const CategoryPage = () => {
...
-                    key={name}
-                    hovered={hovered === name}
-                    category={category}
-                    techName={name}
-                    enabled={enabled}
-                    onClick={() => setTechnology(name)}
-                  />
-                ))}
+                {[...content.data.preferred]
+                  .reverse()
```
Comment (daria-itech) [inline src/containers/CategoryPage/CategoryPage.tsx:293]:
```suggestion
                {content.data.preferred
                  .reverse()
```

Comment (daria-itech) [reply src/containers/CategoryPage/CategoryPage.tsx:293]: do we really need array destructuring here?

File: src/containers/CategoryPage/CategoryPage.tsx [hunk 2]
```diff
@@ -289,44 +289,50 @@ const CategoryPage = () => {
...
-                  <CategoryListItem
-                    key={name}
-                    hovered={hovered === name}
-                    category={category}
-                    techName={name}
-                    enabled={enabled}
-                    onClick={() => setTechnology(name)}
-                  />
-                ))}
+                {[...content.data.preferred]
```
Comment (daria-capco) [inline src/containers/CategoryPage/CategoryPage.tsx:292]: Do we really need array destructing here?
```suggestion
                {content.data.preferred
```

**Review by daria-capco — CHANGES_REQUESTED** [review]

File: src/containers/CategoryPage/CategoryPage.tsx (hunk 2 above)
Comment (loka-capco) [reply src/containers/CategoryPage/CategoryPage.tsx:292]: The `content.data.*` array is used in both the <CategoryListItem> and <CategoryRadar> but they were in the reverse order. The use of destructuring is to make a copy of the array and ensure the reverse() only applied on one side. In terms of performance, it's definitely not a pretty solution. However, the site is static generated so performance impact is negligible.

Comment (daria-capco) [reply src/containers/CategoryPage/CategoryPage.tsx:292]: makes sense

**Review by daria-capco — APPROVED** [review]

**Pull request merged**